src/types/file.types.ts

```diff
@@ -89,10 +89,14 @@ interface License extends BackendType {
   link: string;
```

Comment:
Se litt ekstra på endringene for typer her og for publication. Syns det ble litt rart med en egen `PublicationFileSet`, men fant ikke noen bedre måte å gjøre det på hvor vi kan bruke samme type for argument på `createPublication()`

Reply:
synes ikke dette er så ille. kunne jo brukt `{fileSet: FileSet}` som input i `createPublication`, men synes det er helt ok med en egen type også. up to you!